js/src/components/google-ads-account-card/claim-account-button.js

```diff
@@ -42,6 +42,10 @@ const ClaimAccountButton = ( { onClick = noop, ...restProps } ) => {
 		onClick( event );
 	};
 
+	if ( ! hasFinishedResolution ) {
+		return null;
+	}
```

Comment:
While viewing #2651, discovered that if clicking fast enough, it may open a blank page. Checking `hasFinishedResolution` can not ensure that the `inviteLink` is already available.

https://github.com/user-attachments/assets/79ee9cbf-8006-4d04-81d6-1ca39b439004

Also the claim button will become continually locked on "Waiting..." and it will not be able to open the pop-up window via the button again unless refresh the webpage.

https://github.com/woocommerce/google-listings-and-ads/blob/972f45238d9a780563ee35e1becce4f6d2573222/js/src/components/google-combo-account-card/claim-ads-account/claim-ads-account.js#L49-L53

Reply:
@eason9487 I've added the loading indicator if `inviteLink` is `null`. It should fix the issue you noted.

Reply:
This approach was replaced, based on [this observation](https://github.com/woocommerce/google-listings-and-ads/pull/2644#discussion_r1835815402).

Reply:
> Also the claim button will become continually locked on "Waiting..." and it will not be able to open the pop-up window via the button again unless refresh the webpage.

I've addressed this in [d074504](https://github.com/woocommerce/google-listings-and-ads/pull/2651/commits/d074504253e2c10c91bf944f062bcd51efb80727) of #2651.